src/components/SummaryPanel/ErrorRatePieChart.tsx

```diff
@@ -0,0 +1,30 @@
+import * as React from 'react';
+
+import { PieChart } from 'patternfly-react';
+
+type ErrorRatePieChartPropType = {
+  percentError: number;
+};
+
+export class ErrorRatePieChart extends React.Component<ErrorRatePieChartPropType, {}> {
+  render() {
+    const successRate: number = 100 - this.props.percentError;
```

Comment:
The data in the graph is not going to be percentage, its going to be a fraction "error rate". So the data we put in this component won't be a percentage value. Right @jshaughn ?

Reply:
We should probably just merge this so people aren't stepping on each other.